YvesIraguha/lost-and-found-api
# #171923628 Set-up Eslint

### What does this PR do?
- Set up Eslint
#### Description of Task to be completed?
- enable code linting
- enable auto formating
- rename database variables
#### How should this be manually tested? 
- clone the repo and checkout the branch `ch-setup-eslint-#171923628`
- run `npm i` to install dependencies
- update your `config/congi.js` file with correct variables from `.env` file 
- copy the content of `.env.sample` file and fill it with correct credentials. 
- make a change in some of the `.js` files and you should see prettier working;
#### Any background context you want to provide?
For the autoformatting to work you should install some extensions in your vs code, namely;m`Prettier - Code Formatter`, and `Eslint`
#### What are the relevant pivotal tracker stories?
 #171923628
